packages/odyssey-react-mui/src/Autocomplete.tsx

```diff
@@ -204,6 +226,7 @@ const Autocomplete = <
   getIsOptionEqualToValue,
   testId,
   translate,
+  isVirtualized = false,
```

Comment:
Small non-blocking nit-pick that I won't point out again. If we could alphabetize props, it will help us see them when skimming the code.

I'd like us to have an ESLint rule to auto-fix this, but we don't nor have we prioritized anything like that. It's just something we're doing manually.

Reply:
Thanks for pointing it out - I'll fix this here and make a ticket for us to address this in the future.

Reply:
Fixed.